.pipeline/scripts/set-release-versions.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import os
+import re
+import sys
+from xml.etree import ElementTree
+
+class _CommentedTreeBuilder(ElementTree.TreeBuilder):
+    def __init__(self, *args, **kwargs):
+        super(_CommentedTreeBuilder, self).__init__(*args, **kwargs)
+
+    def comment(self, data):
+        self.start(ElementTree.Comment, {})
+        self.data(data)
+        self.end(ElementTree.Comment)
+
+def _update_readme(sdk_version):
+    _update_file("README.md", r'(maven_central-)(.+)(-blue\.svg)', r'\g<1>%s\g<3>' % sdk_version)
+    _update_file("README.md", r'(a:sdk-core%20AND%20v:)(.+\))', r'\g<1>%s)' % sdk_version)
+
+def _update_pom_files(sdk_version):
+    _update_version_tags(sdk_version)
+
+def _update_version_tags(sdk_version):
```

Comment:
(Minor)

Not sure why delegating to another method. What's the point of having `update_pom_files`?

Reply:
This is straight from the Cloud SDK, should I remove it?

Reply:
Nevermind, I realize this is a recursive function. Hm..